src/main/java/frc/robot/subsystems/shooter/ShooterIO.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ShooterIO {
+  
+  @AutoLog
+  public static class ShooterIOInputs {
+    
+    // Left Shooter Motor Inputs
+    double shootMotorLeftStatorCurrentAmps = 0.0;
+    double shootMotorLeftSupplyCurrentAmps = 0.0;
+    double shootMotorLeftVelocityRPS = 0.0;
+    double shootMotorLeftReferenceVelocityRPS = 0.0;
+
+    // Right Shooter Motor Inputs
+    double shootMotorRightStatorCurrentAmps = 0.0;
+    double shootMotorRightSupplyCurrentAmps = 0.0;
+    double shootMotorRightVelocityRPS = 0.0;
+    double shootMotorRightReferenceVelocityRPS = 0.0;
+
+    // Angle Motor Inputs
+    double angleMotorStatorCurrentAmps = 0.0;
+    double angleMotorSupplyCurrentAmps = 0.0;
+    double angleMotorVelocityRPS = 0.0;
+    double angleMotorReferenceVelocityRPS = 0.0;
+    double angleEncoderAngle = 0.0;
```

Comment:
Specify units (e.g., degrees, radians, rotations). We usually seem to best understand degrees....